### DISCUSSION TITLE
Energie Screen

### DISCUSSION THREAD

**Discussion by Ploegyde** [Blackymas/NSPanel_HA_Blueprint#228] [Ideas]
Would it be possible to  get the HA Energie Screen visable on the panel?
![image](https://user-images.githubusercontent.com/100301784/213981101-70d156be-fc19-4f85-ac54-9845c325f74e.png)

**Comment by Blackymas** [comment]
hi
is accepted as feature requests so it should be implemented in the future :)

https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/168

**Comment by edwardtfn** [comment]
- #168
